ros/src/twist_controller/twist_controller.py

```diff
@@ -86,7 +86,7 @@ def control(self,
                 if -accel < self.brake_deadband:
                     accel = 0.0
                 
-                throttle, brake = 0, -accel * self.brake_car_factor
+                throttle, brake = 0, 20000 # (-accel * self.brake_car_factor) * 150.0
```

Comment:
so the standard torque formula did not seem to produce good braking behaviour?

Reply:
Nope, even with the multiplicative factor the brake didn't worked as expected. The car still moved slowly.